runtime/manual/advanced/migrate_deprecations.md

```diff
@@ -12,11 +12,96 @@ will continue to be updated until the launch of 2.0.
 
 :::
 
-## API changes and deprecations
+## Subcommands
 
-The following APIs have changed or deprecated between Deno 1.x and 2.x, and will
-be removed in 2.x - guidance for migrating to a newer set of APIs is provided
-for each impacted API below.
+### deno bundle
+
+The `deno bundle` command has been removed. We recommend using
+[`esbuild`](https://esbuild.github.io/) together with
+[`esbuild-deno-loader`](https://jsr.io/@luca/esbuild-deno-loader).
+
+```ts
+import * as esbuild from "npm:esbuild";
+import { denoPlugins } from "jsr:@luca/esbuild-deno-loader";
+
+const result = await esbuild.build({
+  plugins: [...denoPlugins()],
+  entryPoints: ["https://deno.land/std@0.185.0/bytes/mod.ts"],
+  outfile: "./dist/bytes.esm.js",
+  bundle: true,
+  format: "esm",
+});
+
+esbuild.stop();
+```
+
+### deno cache
+
+The `deno cache` command has been merged into the `deno install` command under
+the `--entrypoint` option.
+
+```diff
+- deno cache main.ts
++ deno install --entrypoint main.ts
+```
```

Comment:
Actually, `deno cache` is being deprecated in Deno 2, not removed. I'll remove this section.

Reply:
We should not use `deno cache` anymore in any docs - it's getting deprecated, but we don't want to advertise it in any way.

Reply:
Removed

Reply:
@iuioiua Why did you remove it? We _do_ want to move people over to `deno install --entrypoint` from `deno cache`

Reply:
Sorry. Misunderstood. Added back now.